### ISSUE TITLE
Release new version 1.0.1

### ISSUE THREAD

**Issue by dilip-dhankecha** [smartSenseSolutions/smartsense-java-commons#15]
This release includes the following updates to the library:

```
1. Added dynamic projection
2. Improved handling of enum values
```

**Comment by dilip-dhankecha** [comment]
For more details need to refer below PR,

1. [Dynamic projection](https://github.com/smartSenseSolutions/smartsense-java-commons/issues/11)
2. [Enum Value Support](https://github.com/smartSenseSolutions/smartsense-java-commons/issues/14)

**Comment by dilip-dhankecha** [comment]
New release published with version 1.0.1.

https://mvnrepository.com/artifact/com.smartsensesolutions/commons-dao/1.0.1
